core/src/types/version.rs

```diff
@@ -3,7 +3,7 @@ use serde::{Serialize, Serializer, Deserialize, Deserializer, Error};
 use serde::de::Visitor;
 
 /// Protocol Version
-#[derive(Debug, PartialEq, Clone)]
+#[derive(Debug, PartialEq, Clone, Copy)]
 pub enum Version {
```

Comment:
Shouldn't this contain other versions? (Does spec say if `{"jsonrpc":"1.0", "method"....}` is a valid request?)